config/locales/fr.yml

```diff
@@ -8,10 +8,15 @@ fr:
     Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir
     et trouver des informations sur les gems disponibles. Contribuez et améliorez
     ce site avec nous !
-  footer_sponsors_html:
-  footer_join_rt_html:
+  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat avec une grande communauté Ruby.
+    <a href="https://www.fastly.com/">Fastly</a> fournisseur de bande passante et du support CDN,
+    <a href="https://www.rubycentral.org/">Ruby Central</a> qui couvre les frais d'infrastructure, et
+    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> qui finance le développement et le travail d'ops.
```

Comment:
`le travail d'ops`  is there a more formal term for this? I'm guessing ops is operations but it's still not clear to me.

Reply:
I'd suggest: `et l'administration des serveurs.`